schema/data/prod/form_json/production.json

```diff
@@ -87,7 +90,11 @@
   "uiSchema": {
     "ui:add-text": "Add a Product",
     "ui:remove-text": "Remove Product",
+    "ui:field": "ProductsArrayField",
     "items": {
+      "ui:hasRemove": {
+        "isFalse": "isMandatory"
+      },
```

Comment:
Hidden field helps us [override the `hasRemove` button boolean property](https://github.com/bcgov/cas-ciip-portal/pull/1679/files#diff-1c740ccc67b9a1928566d1bbe1c2c266c397b5ba4ad53f404fde5614a6a0eb5aR19-R28) for array fields on a per-item basis.